src/renderer/widgets/SendAssetModal/ui/components/TransferForm.tsx

```diff
@@ -60,9 +74,18 @@ export const TransferForm = ({
   onSubmit,
   onChangeAmount,
   onTxChange,
+  onDestinationChainChange,
+  onDestinationChange,
   feeIsLoading,
   fee,
   deposit,
+  destinations,
+  xcmFee,
```

Comment:
Maybe it's a bit better to use a single `xcmParams` instead of many xcm params one by one?